tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,23 @@
+function sortCountries(a, b) {
+  if (a[1] === b[0]) {
+    return -1;
+  }
+  if (a[0] === b[1]) {
+    return a[0];
+  }
+  return 0;
+}
+
+function followTheSpy(routes) {
+  return routes.sort(sortCountries).reduce((sum, item, index) => {
```

Comment:
Returning chaining functions with a bunch of complex logic is a bad practice. Please separate it in different lines